oggsound/CMakeLists.txt

```diff
@@ -41,6 +41,15 @@ else()
     add_definitions(-DOGGSOUND_THREADED=0)
 endif()
 
+# OpenAL Soft Extensions
+option(HAVE_ALEXT "Enable OpenAL Soft Extensions (alext.h)" OFF)
```

Comment:
one could probably set this automatically with CHECK_INCLUDE_FILES:
https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/How-To-Write-Platform-Checks

however, are there any OpenAL implementations we care about, besides OpenAL soft? If not, we could simply assume that alext.h exists.